.github/workflows/release.yaml

```diff
@@ -59,19 +61,25 @@ jobs:
         with:
           fetch-depth: ${{ env.FETCH_DEPTH }}
 
+      - name: Install toolchain dependencies
+        if: matrix.container == 'ubuntu:18.04'
+        run: apt-get update && apt-get install -y build-essential curl
+
       - name: Install Rust toolchain
-        run: |
-          rustup update --no-self-update stable
-          rustup target add ${{ matrix.target }}
-          rustup component add rust-src
+        uses: actions-rs/toolchain@v1
```

Comment:
Is this actually needed? We were using it before, but it's unmaintained and doesn't do much anyway.

Reply:
the container needs the toolchain explicitly installed as ubuntu:18.04 doesn't come with rustup

Reply:
Oh, right. I think it's better to install it from rustup.rs, though.

Reply:
good idea. i took a page from [dtolnay/rust-toolchain](https://github.com/dtolnay/rust-toolchain/blob/master/action.yml#L44-L45)'s install script which installs from rustup.sh.

Reply:
Do we also want `--profile minimal` there?

Reply:
sure, that's in. [seems to be working](https://github.com/SpencerSharkey/rust-analyzer/runs/8290326299?check_suite_focus=true).

Reply:
Sorry, I think we also need `--components rust-src` because otherwise the `analysis-stats` down below will fail.

Reply:
Ah, never mind, that's done just below.